connectors/services/job_scheduling.py

```diff
@@ -88,9 +88,19 @@ async def _schedule(self, connector):
             raise DataSourceError(msg)
 
         source_klass = get_source_klass(self.source_list[connector.service_type])
+        data_source = source_klass(connector.configuration)
+        data_source.set_logger(connector.logger)
+
+        connector.log_debug("Validating configuration")
+        try:
+            data_source.validate_config_fields()
+            await data_source.validate_config()
```

Comment:
This validates the config of each connector, we can validate it together with this code block:
https://github.com/elastic/connectors/blob/17a76e99c2b3fc0151ba5c65eceb541f84dc6b2f/connectors/services/job_scheduling.py#L91-L97

Reply:
I'm sorry, I don't understand the comment, can you elaborate more?

Reply:
Ignore me, I didn't see you moved the below code out of the if branch
https://github.com/elastic/connectors/blob/17a76e99c2b3fc0151ba5c65eceb541f84dc6b2f/connectors/services/job_scheduling.py#L92-L93